src/panels/lovelace/cards/hui-sensor-card.js

```diff
@@ -85,9 +81,9 @@ class HuiSensorCard extends EventsMixin(LitElement) {
             ${
               _line
                 ? svg`
```

Comment:
Funny, prettier doesn't format HTML inside things tagged with `svg`

Reply:
I updated this so everything looks like prettier handles html tag

Reply:
Actually I just had the Idea to change SVG to HTML format that and change it back.